ci/test_cpp.sh

```diff
@@ -27,7 +27,11 @@ rapids-logger "Begin cpp tests"
 cmake -S testing -B build
 
 cd build
+
+EXITCODE=0
+trap "EXITCODE=1" ERR
+set +e
```

Comment:
Add newline before calling `ctest`.
```suggestion
set +e

```